docs/document/content/user-manual/shardingsphere-jdbc/special-api/sharding/hint.cn.md

```diff
@@ -125,3 +127,46 @@ try (HintManager hintManager = HintManager.getInstance();
     }
 }
 ```
+
+### 使用 Hint 路由至任意指定数据库
+
+#### 使用手动编程的方式
+
+##### 获取 HintManager
+
+与基于 Hint 的数据分片相同。
+
+##### 设置路由至任意指定数据库
+
+- 使用 `hintManager.setDataSourceName` 设置数据库名称。
+
+##### 完整代码示例
+
+```java
+String sql = "SELECT * FROM t_order";
+try (HintManager hintManager = HintManager.getInstance();
+     Connection conn = dataSource.getConnection();
+     PreparedStatement preparedStatement = conn.prepareStatement(sql)) {
+    hintManager.setDataSourceName("ds_0");
+    try (ResultSet rs = preparedStatement.executeQuery()) {
+        while (rs.next()) {
+            // ...
+        }
+    }
+}
+```
+
+#### 使用 SQL 注释的方式
+
+##### 使用规范
+- 需要开启解析注释的配置
+- 注释格式只支持 `/* */`
+- 注释开头需要包含 `ShardingSphere hint:`
```

Comment:
@tuichenchuxin Can we replace ShardingSphere hint with sql hint? sql hint seems to be easier for users to understand.